plugins/org.jkiss.dbeaver.ext.mysql.ui/src/org/jkiss/dbeaver/ext/mysql/ui/config/MySQLCommandChangeUser.java

```diff
@@ -92,6 +92,7 @@ public void afterExecute(DBCSession session, Throwable error)
                     {
                         if (error == null) {
                             getObject().setPersisted(true);
+                            getObject().getDataSource().resetUsers();
```

Comment:
Not clear why to reset already read users here?

Reply:
I reset the cache here because we’ve just added a user, so the cached users is outdated.
Otherwise, getUser(...) and getUsers(...) wouldn’t return the new user.